prize-tool/src/utils.rs

```diff
@@ -0,0 +1,96 @@
+use crate::prize::Winner;
+use solana_core::blocktree::Blocktree;
+use solana_sdk::clock::Slot;
+use solana_sdk::pubkey::Pubkey;
+
+/// Returns an ordered list of slots for the blockchain ending with `last_block` and starting with
+/// `first_block`
+pub fn block_slots(first_block: Slot, last_block: Slot, blocktree: &Blocktree) -> Vec<Slot> {
```

Comment:
How about `block_chain`?

Reply:
Hahaha, I thought of that too! I didn't suggest it because I thought the term might be a little overloaded...? On the other hand, it does seem to be essentially correct.

Reply:
lol I had it originally but ditched it for the same reason. I think I'll stick with it though since it's the least confusing